File: 2533-bitwise-xor-of-all-pairings/2533-bitwise-xor-of-all-pairings.cpp
class Solution {
public:
    int xorAllNums(vector<int>& nums1, vector<int>& nums2) {
        int k=0;
        if(nums1.size()%2==1){
            for(int i:nums2){
                k^=i;
            }
        }
        if(nums2.size()%2==1){
            for(int i:nums1){
                k^=i;
            }
        }
        return k;
    }
};

File: 2533-bitwise-xor-of-all-pairings/2533-bitwise-xor-of-all-pairings.py
class Solution:
    def xorAllNums(self, nums1: List[int], nums2: List[int]) -> int:
        ans1=0
        ans2=0
        for i in nums1:
            ans1^=i
        for j in nums2:
            ans2^=j
        r=0
        if(len(nums2)%2==1):
            r=r^ans1
        if(len(nums1)%2==1):
            r=r^ans2
        return r
        